chore(deps): bump

dcanadillas opened the pull request:
Update [WorkshopCB/petclinic-kaniko](https://github.com/WorkshopCB/petclinic-kaniko.git) 

Command run was `jx import -u https://github.com/WorkshopCB/petclinic-kaniko.git`
<hr />

Update [WorkshopCB/petclinic-kaniko-jx](https://github.com/WorkshopCB/petclinic-kaniko-jx.git) 

Command run was `jx import -u https://github.com/WorkshopCB/petclinic-kaniko-jx.git --no-draft`

Commit:
chore(deps): bump https://github.com/WorkshopCB/petclinic-kaniko-jx.git

Command run was `jx import -u https://github.com/WorkshopCB/petclinic-kaniko-jx.git --no-draft`

dcanadillas-kubecd commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approvers: 

If they are not already assigned, you can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dcanadillas-kube%2Fenvironment-dcanadillas-cjxd-dev).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dcanadillas-kube/environment-dcanadillas-cjxd-dev/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

Commit:
chore(deps): bump https://github.com/WorkshopCB/petclinic-kaniko.git

Command run was `jx import -u https://github.com/WorkshopCB/petclinic-kaniko.git`
